readline/readline.go

```diff
@@ -191,6 +191,12 @@ func (i *Instance) Readline() (string, error) {
 			buf.ClearScreen()
 		case CharCtrlW:
 			buf.DeleteWord()
+		case CharCtrlZ:
+			if err := UnsetRawMode(fd, termios); err != nil {
```

Comment:
It skips the deferred `UnsetRawMode` because it doesn't return until the process is resumed